packages/terra-table/src/terra-dev-site/doc/guides/CheckMarkTable.jsx

```diff
@@ -0,0 +1,94 @@
+import React, {
```

Comment:
It may be a good idea to add inline comment to this file.

Reply:
Updated.